specification/analysisservices/resource-manager/Microsoft.AnalysisServices/2017-08-01-beta/analysisservices.json

```diff
@@ -607,6 +653,60 @@
         }
       }
     },
+    "GatewayDetails": {
+      "description": "The gateway details.",
+      "type": "object",
+      "properties": {
+        "gatewayResourceId": {
+          "type": "string",
+          "description": "Gateway resource to be associated with the server. Only supported with api versions 2017-07-14 or higher."
```

Comment:
Not sure whether it makes sense to document "Only supported with api versions 2017-07-14 or higher." - this very Swagger is tied to an API version, so it's kind of redundant information. On the other hand, you may have thought about what people read in docs generated from this Swagger (and I'm not sure how explicit API versions are called out in our auto-gened docs). But generally, I haven't seen this pattern before! @ravbhatnagar Guidance?